plugins/module_utils/crypto/support.py

```diff
@@ -201,12 +205,21 @@ def load_certificate(path, content=None, backend='cryptography'):
     except (IOError, OSError) as exc:
         raise OpenSSLObjectError(exc)
     if backend == 'pyopenssl':
-        return crypto.load_certificate(crypto.FILETYPE_PEM, cert_content)
+        if identify_pem_format(cert_content) or der_support_enabled is False:
```

Comment:
I would write it this way around:
```suggestion
        if der_support_enabled is False or identify_pem_format(cert_content):
```
then `identify_pem_format(cert_content)` is only ever called if `der_support_enabled is not False`.